docs/modules/qrdqn.rst

```diff
@@ -0,0 +1,101 @@
+.. _qrdqn:
+
+.. automodule:: sb3_contrib.qrdqn
+
+
+QR-DQN
+======
+
+`Quantile Regression DQN (QR-DQN) <https://arxiv.org/abs/1710.10044>`_ builds on `Deep Q-Network (DQN) <https://arxiv.org/abs/1312.5602>`_
+and make use of quantile regression to explicitly model the distribution over returns.
+
+
+.. rubric:: Available Policies
+
+.. autosummary::
+    :nosignatures:
+
+    MlpPolicy
+    CnnPolicy
+
+
+Notes
+-----
+
+- Original paper: https://arxiv.org/abs/1710.100442
+
+
+Can I use?
+----------
+
+-  Recurrent policies: ❌
+-  Multi processing: ❌
+-  Gym spaces:
+
+
+============= ====== ===========
+Space         Action Observation
+============= ====== ===========
+Discrete      ✔      ✔
+Box           ❌      ✔
+MultiDiscrete ❌      ✔
+MultiBinary   ❌      ✔
+============= ====== ===========
+
+
+Example
+-------
+
+.. code-block:: python
+
+  import gym
+  import numpy as np
+
+  from sb3_contrib import QRDQN
+
+  env = gym.make("CartPole-v1")
+
+  model = QRDQN("MlpPolicy", env, verbose=1)
+  model.learn(total_timesteps=10000, n_quantiles=50, log_interval=4)
```

Comment:
thanks for editing, but this is not how it works, no? (again take a look at TQC example ;) )

Reply:
Sorry about that...